src/formatDate.js

```diff
@@ -8,7 +8,35 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separatorFrom = fromFormat.pop();
```

Comment:
Using `pop` mutates the original `fromFormat` array, which is generally not a good practice as it can lead to side effects if the array is used elsewhere after calling this function. Consider using a different approach that doesn't modify the input arrays.

Reply:
Yes, this method is mutating, but the task conditions do not prohibit modifying the original data, and we do not plan to reuse this function in the code.